### ISSUE TITLE
Shear Mooshrooms

### ISSUE THREAD

**Issue by FlyingPikachu** [FlyingPikachu/Quests#272]
This is a personal reminder to either expand the "Shear Sheep" objective to include Mooshrooms, or to add a separate "Shear Mooshrooms" objective.

**Comment by Midnight9746** [comment]
You could just expand the sheer sheep and call it "Sheering" and then do what you have done with the mob hunting where you pick a sheep or mooshroom. I think that adding two separate objectives would be too confusing for some.

I personally like everything of one objective type in one spot.

**Comment by FlyingPikachu** [comment]
Spent a nice chunk of time yesterday trying to add Mooshrooms. Went about changing the "Sheer Sheep" event to "Sheer mobs" and essentially gave it the same functionality of "Tame mobs", plus colors. However, I ran into some roadblocks, namely difficulty keeping the new functionality compatible with previously-made shearing quests (as not to break anyone's server) and, more significantly, the fact that Bukkit/Spigot doesn't appear to provide a way to check if a particular EntityType is shear-able (unlike tamed mobs).  I also found out the Snowman class is shear-able, and at that point I decided to forego this task. https://www.spigotmc.org/threads/check-if-an-entitytype-is-shear-able.304213/

I agree with @Midnight9746 that separate objectives for Mooshroom and Snowman would be excessive, so I think this is best left to a custom objective. Oh well!
